### ISSUE TITLE
Help with setup

### ISSUE THREAD

**Issue by nfstange** [SR2k/homebridge-new-widetech-internet-dehumidifier#2]
<img width="874" alt="Screen Shot 2021-12-24 at 4 33 46 PM" src="https://user-images.githubusercontent.com/52550843/147373560-2a87a320-e591-4426-a5d6-96ae81ad6426.png">
I am trying to setup this plugin and I have acquired my devices token and IP address and after configuring the config file it won't appear in Homekit.

**Comment by SR2k** [comment]
You should use the exact name `NewWidetechInternetDehumidifier` with no spaces.

**Comment by nfstange** [comment]
Hi I tried changing that and still no luck
<img width="688" alt="Screen Shot 2021-12-27 at 8 01 33 AM" src="https://user-images.githubusercontent.com/52550843/147478898-b6296144-5594-43d9-a155-218875c25d9c.png">
<img width="1241" alt="Screen Shot 2021-12-27 at 8 00 21 AM" src="https://user-images.githubusercontent.com/52550843/147478918-b0f5a1a8-d770-4a69-bffa-7b409ed06d45.png">

**Comment by SR2k** [comment]
Could you please run HomeBridge under debug mode using `homebridge -I -D` and copy all the associated logs here?

**Comment by nfstange** [comment]
> Could you please run HomeBridge under debug mode using `homebridge -I -D` and copy all the associated logs here?

Hi could you assist me with that. i’m not to familiar with homebridge i typically use HOOBS.

**Comment by SR2k** [comment]
> > Could you please run HomeBridge under debug mode using `homebridge -I -D` and copy all the associated logs here?
> 
> Hi could you assist me with that. I’m not too familiar with homebridge I typically use HOOBS.

Of course. I need a little bit more info about your environment. How did you get your HomeBridge installed and booted?

**Comment by nfstange** [comment]
Hi sorry for the late reply! I'm running homebridge on my mac mini.
